benchmarks/result_analyzer.py

```diff
@@ -58,13 +69,92 @@ def run(self):
         "outputs_file": pd.Series(dtype="str"),
     })
     for file in jsonl_files:
-      metric_df = self.extract_metrics(file, metric_df)
+      metric_df = self.extract_metrics_csv(file, metric_df)
 
     # additional processing of the metric_df can be done here
 
     self.export_metric_report(metric_df)
 
-  def extract_metrics(self, file, metric_df):
+  def get_calculated_metrics(self, d, tmp):
```

Comment:
Could this method args be made more readable? It's okay to use `tmp` in the method call, but in method definition, it's better to reflect the actual object that is passed.

In this case `tmp` is a dict read from a json string representing the raw results of one experiment.

Reply:
Thanks, I will make the update. By the way, we will drop this PR and instead, merge this into main branch directly.